### PR TITLE
Switch delete button when pressed

### PR DESCRIPTION
**Pull request by misteu**
### What does it do
- adds method for styling delete button
- changes delete button's style when pressing similar to native delete button
- see video of the feature attached here: https://github.com/scribe-org/Scribe-iOS/issues/204#issuecomment-1260087859

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Emojis for the following are chosen based on [gitmoji](https://gitmoji.dev/).
 
 - The annotation colors have been changed to match the new backgrounds.
 - The App Store images have been updated to reflect autosuggest data based on Wikipedia.
+- Delete key features pressed state style similar to the native keyboard
```
Comment (andrewtavis) [inline CHANGELOG.md:28]: Thanks for also editing the changelog! Nice to get contributions to this 😊

Comment (andrewtavis) [inline CHANGELOG.md:28]: Thanks for also editing the changelog! Nice to get contributions to this 😊

**Review by andrewtavis — APPROVED** [review]
Thanks so much for this, @misteu! Really a nice first issue to do here. Really appreciate the changelog edit and adding the doc string to your function :)

**Comment by andrewtavis** [comment]
Let me know if for some reason this isn't being counted towards Hacktoberfest for you. Is my first year participating/maintaining, so just want to make sure you get the credit you deserve :)

Thanks again!

**Comment by misteu** [comment]
Hi! Just checked my hacktoberfest profile, it was not counted yet.
You need to set the repository's topic to `hacktoberfest` or label the PR `hacktoberfest-accepted`

grafik

I think you can still change it. I guess they are checking it every few hours or so.

**Comment by andrewtavis** [comment]
@misteu, I just added the `hacktoberfest` tag :) Let me know if it works!

**Comment by andrewtavis** [comment]
@misteu, also just added the label :)

**Pull request merged**